deepmd/pt/model/model/ener_hess_model.py

```
@@ -0,0 +1,145 @@
+    BaseModel,
+)
+from .dp_model import (
+    DPModelCommon,
+)
+from .make_model import (
+    make_model,
+)
+from .make_hessian_model import (
+    make_hessian_model,
+)
+import numpy as np
```

Ruff

`numpy` imported but unused

Remove unused import: `numpy`

(F401)